nuxt.config.ts

```diff
@@ -8,5 +8,13 @@ export default defineNuxtConfig({
   content: {
     // https://content.nuxtjs.org/api/configuration
   },
-  ssr: false
+  app: {
+    baseURL: '/NEBULA/'
```

Comment:
What happens to the framework when it's used as a template, and the repository has a different name? Does it need to be changed here?

Reply:
After checking some docs: seems that way :)
Perhaps add a note in Readme about this; when reusing the project, this needs to be adjusted. We can see if we can automate this later, but for now, let's just keep track.

Reply:
Yes I had the same thought... ghpages seem generally to be hosted at a url that looks like `https://[Org or User].github.io/[Repo name]/`. In my case, it was hosted at <https://raar1.github.io/NEBULA/> but on another project we have e.g. <https://fair-workflows.github.io/nanopub/>. In practice you can choose to host on another domain if you like, if you own it (I think)

If we don't set the baseurl, it defaults to '/', meaning all links will be relative just to the domain name (e.g. just `https://raar1.github.io/`) which is obviously annoying.

I think this is hard to avoid, since nuxt does not know where you will be hosting your static site when it generates all the links. However, perhaps there is some way (passing something to the deploy workflow?) to do this automatically? Not sure. We can open a new issue to investigate if you like